eucadeploy/componentdeployer.py

```diff
@@ -23,9 +24,10 @@ def __init__(self, password, environment_file='environment.yml',
         with hide(*self.hidden_outputs):
             local('if [ ! -d eucalyptus-cookbook ]; then '
                   'git clone '
-                  'https://github.com/eucalyptus/eucalyptus-cookbook;'
-                  'fi')
+                  '{0} eucalyptus-cookbook;'
+                  'fi'.format(cookbook))
             local('cd eucalyptus-cookbook; git checkout {0};'.format(cookbook_branch))
+            local('git pull')
```

Comment:
We could use the fabric cd context manager: http://docs.fabfile.org/en/latest/api/core/context_managers.html#fabric.context_managers.cd